site/en/reference/cli/concepts/coaps.md

```diff
@@ -0,0 +1,143 @@
+# Test Connectivity to a Constrained Application Protocol Secure (CoAPS) Resource Using the OT CLI
+
+OpenThread offers both CoAPS server and client functionality, enabling devices
```

Comment:
CoAP is well-known abbreviation. But "CoAPS" is not. Suggest using `secure CoAP` instead of `CoAPS` in the docs here and below?